arbitrum-docs/how-arbitrum-works/06-optimistic-rollup.mdx

```diff
@@ -27,11 +29,21 @@ Among optimistic rollups, the most important design decision is how to resolve d
 
 There are basically two choices: interactive proving, or re-executing transactions. Arbitrum uses interactive proving, which we believe is more efficient and more flexible. Much of the design of Arbitrum follows from this fact.
 
+::: note
+
+Zero-knowledge rollups are able to avoid this situation entirely by using ZK proofs to prove directly to the parent chain that the posted result is correct.
```

Comment:
```suggestion
Zero-knowledge Rollups are able to avoid this situation entirely by using ZK proofs to prove directly to the parent chain that the posted result is correct.
```